packages/taquito-michelson-encoder/src/taquito-michelson-encoder.ts

```diff
@@ -3,16 +3,16 @@
  * @module @exodus/taquito-michelson-encoder
  */
 
-export * from './schema/storage';
-export * from './schema/parameter';
-export * from './schema/view-schema';
-export * from './schema/error';
-export * from './schema/types';
-export { Semantic, SemanticEncoding, BigMapKeyType } from './tokens/token';
-export * from './errors';
+export * from './schema/storage.js';
+export * from './schema/parameter.js';
+export * from './schema/view-schema.js';
+export * from './schema/error.js';
+export * from './schema/types.js';
+export type { Semantic, SemanticEncoding, BigMapKeyType } from './tokens/token.js';
```

Comment:
@ChALkeR looks like jest is more lenient here - exodus-test required me to refactor `export` -> `export type`

Reply:
This is correct

Otherwise it's not possible to transform this file without looking at the whole project tree

These re-exports should be removed by the ts -> js build process, because this code would be invalid in js -- there are no `Semantic, SemanticEncoding, BigMapKeyType` in transformed `./tokens/token.js`

Ideally we should always use `(import|export) type ... from ` for importing or re-exporting types

Reply:
agreed!

Reply:
See also https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax which forbids that
Which we should have enabled everywhere